docs/quick-starts/feature-flag-quickstart.md

```diff
@@ -33,19 +33,19 @@ Give the flag a descriptive human readable name and create variations for each v
 
 ![Feature gate 1](/img/feature-flagging/feature-flag-qs-1.png)
 
-### 3. Create an experiment allocation
+### 3. Create a Feature Gate allocation
 
 After creating the flag, switch into the Test environment:
 
 ![Feature gate 2](/img/feature-flagging/feature-flag-qs-2.png)
 
-Now that you're in the Test environment, add an Experiment allocation to your flag. If you want to force certain segments or users into one variant, you can also add a Feature Gate allocation. You can read more about using Eppo for Feature Gates [here](/feature-flagging/feature-gates).
+Now that you're in the Test environment, add a Feature Gate allocation to your flag. If you want to force certain segments or users into one variant, you can also add a Feature Gate allocation. You can read more about using Eppo for Feature Gates [here](/feature-flagging/feature-gates).
 
-![Feature gate 3](/img/feature-flagging/feature-flag-qs-3.png)
+![Feature gate 3](/img/feature-flagging/feature-gate-setup-1.png)
 
 For this example, we will assign all users to the experiment. If you want to target specific users, you can add targeting rules to the allocation. You can read more about targeting [here](/feature-flagging/targeting).
 
-![Feature gate 4](/img/feature-flagging/feature-flag-qs-4.png)
+![Feature gate 4](/img/feature-flagging/feature-gate-setup-3.png)
 
 ### 4. Connect a logging function to the Eppo SDK
```

Comment:
We can skip logging for the sake of this guide.

Reply:
If so, we should update the initialization code to not have an event logger

Reply:
done